### ISSUE TITLE
Auto Memory Optimisation By Another Timer 

### ISSUE THREAD

**Issue by modaresimr** [CodeDead/MemPlus#12]
Currently, auto optimization threshold is working where the ram usage is updating (by any cases such as after clearing memory, by update timer, etc.) [source](https://github.com/CodeDead/MemPlus/blob/f1d543e1363f06341271b273dc6a8672aa49edbb/MemPlus/Business/RAM/RamController.cs#L491)
Currently, if the required memory (the memory after clearing memory) is more than the threshold it will be called frequently.

I think it would be better to have a separated adjustable timer for auto optimisation by threshold.

Thanks for your great work

**Comment by CodeDead** [comment]
Indeed, this is mentioned in [the manual](https://codedead.com/Software/MemPlus/help.pdf) (page 6). Optimizing too often is detrimental to performance. Something like a timer would be an excellent solution to this problem.

I'm keeping this open for a future update so that this will be implemented. Thanks for the suggestion!
